tests/core/test_ctors_no_main.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2019 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <emscripten.h>
```

Comment:
I think it might be better to have a testcase that does work in wasi runtimes, for more coverage? The test could be a printf in a global constructor for example, and avoid EM_ASM.

Reply:
Done